src/main/java/gift/ProductRepository.java

```diff
@@ -0,0 +1,64 @@
+package gift;
```

Comment:
```suggestion
package gift.repository;
```
여기두 패키지명 변경.
이렇게 패키지를 분리를 해놔야 나중에 유지보수 관점에서 좋고 추후에 배울 AOP를 적용하기 위해서 관리가 더 용이하답니다 👍